### PR TITLE
Feature/df generator/speak leash to df

### LINKED ISSUE #4: SpeakLeash and pandas
Create an example of how to quickly put data from the SpeakLeash into a data frame

### PR DESCRIPTION
**Pull request by przemekboruta**
Resolves #4

### CODE REVIEW

File: example_4_pandas/charts.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Generate .html charts from the dataset
```
Comment (IgorTest19) [inline example_4_pandas/charts.py]: For module docstrig, the text should be contained in triple brackets, like::
`"""Generate .html charts from the dataset"""`

File: example_4_pandas/charts.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+import os
+import pandas as pd
+import plotly.express as px
+
+from speakleash import Speakleash
+
+save_path = "imgs/"
```
Comment (IgorTest19) [inline example_4_pandas/charts.py]: Constant names could be written in UPPER_CASE naming style:
`SAVE_PATH = "imgs/"`

File: example_4_pandas/charts.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+
+class Chart:
```
Comment (IgorTest19) [inline example_4_pandas/charts.py]: Descriptions (docstrings) should be added to the class and methods.

**Review by IgorTest19 — CHANGES_REQUESTED** [review]

**Comment by Samox1** [comment]
Only suggestion - for clarity, leave only the example of loading data into a pandas DataFrame and making a separate folder for the demonstration of charts from charts.py

**Comment by IgorTest19** [comment]
> The only suggestion - for clarity, leave only the example of loading data into pandas DataFrame and create a separate folder for demonstrating charts with charts.py.

This is a good suggestion, but in order not to make things too complicated, we can leave both main.py and charts.py files in one directory and create a short explanation in the readme.md file with the purpose/usage of both files. This pull request was created for the issue: [SpeakLeash and pandas #4](https://github.com/speakleash/speakleash-examples/issues/4)
which was about creating a data frame based on the Speakleash data. More complex examples should be done in the another issue, as this issue resolves a fairly basic case.

File: example_4_pandas/charts.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
+import os
```
Comment (IgorTest19) [inline example_4_pandas/charts.py]: Add module docstring (short description)

File: example_4_pandas/main.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+import os
+import pandas as pd
+import pickle
+
+
+from speakleash import Speakleash, Reader
+
+
+def get_reader(project_name: str, speakleash_class: Speakleash) -> Reader.stream_data:
```
Comment (IgorTest19) [inline example_4_pandas/main.py]: add short descriptions to these functions

**Review by IgorTest19 — APPROVED** [review]

File: example_4_pandas/main.py (hunk 5 above)
Comment (IgorTest19) [reply example_4_pandas/main.py]: Done

File: example_4_pandas/charts.py (hunk 4 above)
Comment (IgorTest19) [reply example_4_pandas/charts.py]: Done

**Review by IgorTest19 — APPROVED** [review]

**Pull request merged**